### ISSUE TITLE
peerDep warning during terra-framework npm install

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-framework#547]
# Issue Description
When running npm install in the project, the following peerDependency warning is being logged:
```
npm WARN terra-doc-template@1.23.0 requires a peer of terra-base@^3.12.0 but none is installed. You must install peer dependencies yourself.
```

terra-doc-template 2.0.0 has been released which has a peer on terra-base ^4.0.0. Need to figure out where terra-doc-template@1.23.0 is coming from and update it to terra-doc-template ^2.0.0

**Comment by bjankord** [comment]
I temporarily disabled the package-lockfile and ran npm install, then ran [npm-why](https://www.npmjs.com/package/npm-why) to figure out why we are getting terra-doc-template@1.23.0
Looks like this is coming from terra-dev-site depending on terra-app-delegate.

`terra-framework > terra-dev-site > terra-app-delegate > terra-doc-template@1.23.0`.

Related issue: https://github.com/cerner/terra-dev-site/issues/138

**Comment by bjankord** [comment]
This has been resolved with the release of terra-dev-site 3.4.0
